src/views/StoriesList.vue

```diff
@@ -85,6 +68,72 @@ async function getStories() {
   }
 }
 
+async function loadDropdownData() {
+  try {
+    const [charactersResponse, settingsResponse, languagesResponse, countryResponse] = await Promise.all([
+      CharacterServices.getCharacters(),
+      SettingsServices.getSettings(),
+      LanguagesServices.getLanguages(),
+      CountriesServices.getCountries()
+    ]);
+
+    leadCharacters.value = charactersResponse.data;
+    secondaryCharacters.value = charactersResponse.data;
+    settings.value = settingsResponse.data;
+    languages.value = languagesResponse.data;
+    countries.value = countryResponse.data;
+  } catch (error) {
+    console.log(error);
+    snackbar.value.value = true;
+    snackbar.value.color = "error";
+    snackbar.value.text = "Failed to load dropdown data.";
+  }
+}
+
+async function generateStory() {
+  loading.value = true;
+  try {
+    await StoryServices.addStory(user.value.id, {
+      storyParams: [
+        { key: 'setting', value: newStory.value.setting },
+        { key: 'country', value: newStory.value.country },
+        { key: 'lead character', value: newStory.value.leadCharacter },
```

Comment:
Looking at the dB and models, all we have for story attributes is characters, countries, languages and settings.